src/Model/BoardPackage/Board.java

```diff
@@ -154,5 +154,11 @@ public void sendToJail(Player player) {
         return squaresList.stream().map(squaresList -> (OwnableSquare) squaresList)
                 .filter(ownableSquare -> ownableSquare.getGROUP_NUMBER() == groupId).collect(Collectors.toList());
     }
+
+    public ArrayList<BoardSquare> getLocationByType(int typeID) {
```

Comment:
you also included some other changes that I made while working on the game class. You should get rid of this too.